docs/user-guide.md

```diff
@@ -38,7 +38,13 @@ This guide provides detailed instructions on how to configure your Liberty proje
 
 ### Settings
 
-To ensure that the Liberty Config Language Server starts properly, define the `xml.java.home` property in your VS Code settings or set the system `JAVA_HOME` environment variable to point to Java 17 .
+Liberty Tools will go through a series of checks to locate Java 17 and will alert you when it fails to do so.
+
+Liberty Tools first checks the [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) from the `redhat.java` plugin. If handling multiple Java runtimes causes this requirement to fail, you can define properties in your VS Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings) to help point Liberty Tools to your Java 17 runtime. Liberty Tools will search for `java.jdt.ls.java.home` and `java.home` before checking for the system environment variables `JDK_HOME` and `JAVA_HOME`.
```

Comment:
"If handling multiple Java runtimes causes this requirement to fail" - I'm not sure I follow what this means